packages/common/stakeApy.ts

```diff
@@ -1,22 +1,48 @@
 import { SOL_API_HOST } from './constants';
+import { ErrorWrapper } from '../core/src/utils/errorWrapper';
+import { ERROR_CODE } from '../core/src/constants/errors';
+
+export const STATIC_DEFAULT_APY = 9.35; // TODO think
+
+const { NO_APY_DATA } = ERROR_CODE;
+
+type ApyData = {
+  apy: number;
+  apr: number;
+  intervalPrices: {
+    beginEpoch: number;
+    endEpoch: number;
+  };
+};
 
 type StakeApyResponse = {
   data: {
-    apy: number;
+    lastEpoch: ApyData;
+    twoWeeks: ApyData;
+    oneMonth: ApyData;
+    threeMonth: ApyData;
+    sinceLaunch: ApyData;
   };
 };
 
-export const STATIC_DEFAULT_APY = '5.74'; // TODO think
+export const getMaxApy = (data: StakeApyResponse['data']) => {
+  return Object.values(data).reduce((max, curr) => {
```

Comment:
```suggestion
  return Math.max(...Object.values(data))
```

Reply:
![image](https://user-images.githubusercontent.com/99960252/203860223-79e893b1-fb39-4980-8041-6becb27cb13a.png)
 please look at all function. I need to get link to max apy key, not value. If I need only value, I would write like this:
 
 ```ts
 Math.max(...Object.values(data).map(({apy}) => apy));
 ```